### PR TITLE
Provide a tasklet-free example

### PR DESCRIPTION
**Pull request by jserv**
It is possible that in future tasklets may be replaced by threaded irqs.

### CODE REVIEW

File: lkmpg.tex [hunk 1]
```diff
@@ -1669,6 +1673,11 @@ \subsection{Bottom Half}
 
 \samplec{examples/bottomhalf.c}
 
+It is expected that at some unspecified point in the future tasklets may eventually be deprecated, to be replaced by \textit{threaded irqs} instead.
```
Comment (linD026) [inline lkmpg.tex:1676]: In 2020, the removal of the tasklet API also has been discussed.
See https://lwn.net/Articles/830964/.

Comment (linD026) [reply lkmpg.tex:1676]: Also, the patch of tasklets in 2020 has the same improvement as the `timer_setup`. In other words, the CET needs to be mention too.
See:
* https://github.com/torvalds/linux/commit/12cc923f1ccc1df467e046b02a72c2b3b321b6a2
* https://lwn.net/ml/kernel-hardening/20200716030847.1564131-1-keescook@chromium.org/

**Review by linD026 — CHANGES_REQUESTED** [review]

**Comment by 0xff07** [comment]
I sent a pull request to the `threaded-irq` branch.

**Comment by jserv** [comment]
Close in favor of #255

**Pull request closed without merging**